spec/charms/string_spec.cr

```diff
@@ -2,11 +2,10 @@ require "../spec_helper"
 
 describe "String charm" do
   describe "squish" do
-    it "squishes the text by removing newlines and extra whitespace" do
-      og_string = " foo   bar    \n   \t   boo"
```

Comment:
Can we also leave this in? I dig the new spec, but I'm thinking leave both of them in just for the extra safety. What are your thoughts?